### PR TITLE
Remove analytics-pipeline

### PR DESCRIPTION
**Pull request by macdiesel**
The edx-analytics-pipeline repository is being retired. This PR removes these code bits from devstack. For more info see https://openedx.atlassian.net/browse/DEPR-119

### CODE REVIEW

**Review by jmbowman — APPROVED** [review]
No complaints on the technical side of things, although the PR should have a description.  Should probably also give a little more time for feedback on [DEPR-119](https://openedx.atlassian.net/browse/DEPR-119) before merging.

**Comment by macdiesel** [comment]
I agree. I'll fix that up.

File: docker-compose-analytics-pipeline.yml [hunk 1]
```diff
@@ -1,155 +0,0 @@
-version: "2.1"
-
-services:
-  namenode:
-    image: edxops/analytics_pipeline_hadoop_namenode:${OPENEDX_RELEASE:-latest}
```
Comment (pwnage101) [inline docker-compose-analytics-pipeline.yml:5]: Maybe we should also delete all the logic to build these images? https://github.com/edx/configuration/tree/master/docker/build

**Review by pwnage101 — APPROVED** [review]
Disclaimer: I did not do any grepping, but what I see in this PR looks good.

File: docker-compose-analytics-pipeline.yml (hunk 1 above)
Comment (macdiesel) [reply docker-compose-analytics-pipeline.yml:5]: Good idea!

**Pull request merged**